Dev/GitHub/pr-buddy.30m.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# <xbar.title>Github PR buddy</bitbar.title>
+# <xbar.version>v2.1</bitbar.version>
+# <xbar.author>Luis Almeida Santos</bitbar.author>
+# <xbar.author.github></bitbar.author.github>
+# <xbar.desc>This plugin displays relevant Pull Requests in Github repositories.</bitbar.desc>
+# <xbar.image>https://user-images.githubusercontent.com/71266511/171880002-8f35b14c-16b5-4c0e-a696-a5b4ec1e8cbc.png</xbar.image>
+# <xbar.dependencies>python</bitbar.dependencies>
+
+# <xbar.var>string(USERNAME=""): Your Github username. (ie: luis-santos-teampicnic)</xbar.var>
+# <xbar.var>string(TOKEN=""): A GitHub Personal access tokens with, at least, the repo scope. (https://github.com/settings/tokens/new)</xbar.var>
+# <xbar.var>string(REPOS_TO_CHECK="PicnicSupermarket/jolo,PicnicSupermarket/hiring-experience,PicnicSupermarket/oss-parent,PicnicSupermarket/nepsnowplow,PicnicSupermarket/diepvries,PicnicSupermarket/reactive-support,PicnicSupermarket/employer-statement-generator"): Comma separated list of repositories to check.</xbar.var>
+# <xbar.var>number(MINIMUM_APPROVALS=2): Show the PR number for each line.</xbar.var>
+# <xbar.var>boolean(SHOW_PR_NUMBER=false): A GitHub Personal access tokens with, at least, the repo scope.</xbar.var>
```

Comment:
the description doesn't match the var name?

Reply:
ahhh Thanks! It has been updated! :)